### ISSUE TITLE
Drag and drop SMD gives nothing

### ISSUE THREAD

**Issue by PrincessMargarita** [TiberiumFusion/Smd2Pac#1]
When dragging the SMD, the application closes immediately and there is no output file or json string anywhere, it does not work with any SMD I am trying to drag. none of the lnk file parameters work

Detail: SMD animations decompiled by Crowbar

**Comment by TiberiumFusion** [comment]
Can you send me the SMD file you are using? The file could be invalid, or it could contain data that Smd2Pac does not understand correctly. I would need to inspect it to determine that.

Here's a quick demonstration of what should happen when you drag-and-drop:
![Demo gif](https://github.com/user-attachments/assets/1bc89e29-0adf-40ba-a28d-c7f014782265)

Smd2Pac will print error messages if something goes wrong, but the command window quickly closes when using drag-and-drop. If you want to see the error messages, you should use the command line or create a batch file that includes a `pause` command.
Example batch file:
```bat
Smd2Pac.exe --smd "idle_all_angry.smd"
Smd2Pac.exe --smd "ACT_BOOP.smd" --make-additive-from "idle_all.smd"
Smd2Pac.exe --smd "ACT_BUTTWIGGLE.smd" --make-additive-from "idle_all.smd" --optimize 0 --dump-additive-smd
pause
```

**Comment by PrincessMargarita** [comment]
-[-----     Smd2Pac 1.2.0.2     -----]-

Processing 1 SMD files...

Reading SMD file: "ACT_BOOP.smd" (1/1)
[!!!] Error parsing SMD file [!!!]
    - Invalid SMD data at line 127. Invalid bone pose; bone X translation is not a valid number.
    
  https://www.dropbox.com/s/yhthby9uzj6e9is/ACT_BOOP.smd?dl=1
    
 --make-additive-from" gives the same result
   
UPD: ACT_BUTTWIGGLE gives errors too:

c:\Users\Margarita\Downloads\Smd2Pac.1.2.0.2>Smd2Pac.exe --smd "ACT_BUTTWIGGLE.smd" --make-additive-from "idle_all.smd" --optimize 0 --dump-additive-smd

-[-----     Smd2Pac 1.2.0.2     -----]-
Reading subtraction base pose SMD file: "idle_all.smd"
[!!!] Error parsing SMD file [!!!]
    - Invalid SMD data at line 47. Invalid bone pose; bone X translation is not a valid number.

c:\Users\Margarita\Downloads\Smd2Pac.1.2.0.2>pause

**Comment by TiberiumFusion** [comment]
@PrincessMargarita Thanks for the files and additional information. I've just released a new version (1.2.1.0) that should fix the issue for you. You can download it here: [https://github.com/TiberiumFusion/Smd2Pac/releases/tag/1.2.1.0](https://github.com/TiberiumFusion/Smd2Pac/releases/tag/1.2.1.0)
Please let me know if it solves the issue for you.

The issue was likely regarding decimal parsing rules and mismatched locales. In the new version, Smd2Pac now parses SMD files in their expected locale by default.

**Comment by PrincessMargarita** [comment]
![bandicam 2022-05-22 13-03-29-253](https://user-images.githubusercontent.com/105990827/169687745-460b61e3-ed3e-4471-ab3e-40e6455c2425.jpg)

Problem solved. The exiting folder and file have been created. Thank you very much!
